gn3/computations/biweight.py

```diff
@@ -0,0 +1,26 @@
+
+
+"""module contains script to call biweight mid\
+correlation in R"""
+import subprocess
+import os
+from pathlib import Path
+from typing import List
+
+FILE_PATH = os.path.join(Path(__file__).parent.absolute(), "biweight.R")
```

Comment:
**suggestion:** How about setting this as a variable. Say `BIWEIGHT_RSCRIPT`. And then we could provide that from guix later? For now, we could avail that as a hard path. WDYT?